jenkins-pipelines/oss/kafka_connectors/longevity-kafka-cdc-aws.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'docker',
+    region: 'eu-west-1',
+    availability_zone: 'c',
+
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/kafka/longevity-kafka-cdc-docker.yaml',
```

Comment:
If the config file is suitable also to AWS, why then the file has the "docker" word in it?

Reply:
I'll rename the file

It was before I thought of reusing it to multiple backends